CHANGELOG.md

```diff
@@ -36,6 +69,7 @@
 - with\_driver "aws::us-west-2" block breaking VPC launch in loop [\#369](https://github.com/chef/chef-provisioning-aws/issues/369)
 - ready\_machine should wait for "Windows is Ready to use" message [\#355](https://github.com/chef/chef-provisioning-aws/issues/355)
 - Type Error, Can't Dupe NilClass when creating a load balancer [\#351](https://github.com/chef/chef-provisioning-aws/issues/351)
+- ERROR: bad URI\(is not URI?\): https://\[chef\_server\_url\]/organizations/\[organization\_name\]/data/aws\_eip\_address/assign eip [\#257](https://github.com/chef/chef-provisioning-aws/issues/257)
```

Comment:
Not sure. The way to fix changelog problems is to go to the issue in question and fix the title, though--next time we generate it will go through!